.github/workflows/test.yml

```diff
@@ -1,84 +1,79 @@
 name: CI
 
 on:
-  push:
-    branches: [ v2 ]
-  pull_request:
-    branches: [ v2 ]
+    push:
+        branches: [v2]
+    pull_request:
+        branches: [v2]
 
 permissions:
-  contents: read
+    contents: read
 
 jobs:
-  tests:
-
-    runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        node-version: [14.x, 16.x, 18.x]
-
-    steps:
-      - uses: actions/checkout@v3
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v3
-        with:
-          node-version: ${{ matrix.node-version }}
-      - run: npm ci
-      - run: npm test
-
-  build:
-
-    runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        node-version: [14.x, 16.x, 18.x]
-
-    steps:
-      - uses: actions/checkout@v3
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v3
-        with:
-          node-version: ${{ matrix.node-version }}
-      - run: npm ci
-      - run: npm run build
-
-  lint:
-
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v3
-      - name: Use Node.js 14.x
-        uses: actions/setup-node@v3
-        with:
-          node-version: 14.x
-      - run: npm ci
-      - run: npm run lint:ci
-
-  type-check:
-
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v3
-      - name: Use Node.js 14.x
-        uses: actions/setup-node@v3
-        with:
-          node-version: 14.x
-      - run: npm ci
-      - run: npm run tsc
-
-  coverage:
-
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v3
-      - name: Use Node.js 14.x
-        uses: actions/setup-node@v3
-        with:
-          node-version: 14.x
-      - run: npm ci
-      - run: npm run regex-coverage
+    tests:
+        runs-on: ubuntu-latest
+
+        strategy:
+            matrix:
+                node-version: [16.x, 18.x, 20.x]
```

Comment:
On node version 14.x, I was seeing an error due to `prettier-syncrhonized`. I'm not sure why it wasn't caught in the previous PR.

The issue is, that it was using the `??=` operator [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment) which isn't supported in v14
 
I am not sure how many people are using node v14. The main result of this, is they wouldn't be able to run prettier tests. 


This PR actually changes the support matrix from 14,16,18 -> 16,18,20, but it's really just a suggestion. We have other options